validator_pub_keys/Swiss-Staking.json

```
@@ -0,0 +1,5 @@
+{
+    "validator_name":"Swiss Staking",
+    "validator_address":"cosmosvaloper1y0us8xvsvfvqkk9c6nt5cfyu5au5tww2ztve7q",
+    "stargate_consensus_public_key":"cosmospub1addwnpepqvuzsqcrj42qsrwcqhye0nk62la6eh5dqzahw73p4rs64v997yw3ww0srf3"
+}
```

validator_pub_keys/Swiss-Staking.json
```suggestion
}
```